crates/shared/Cargo.toml

```diff
@@ -58,4 +58,5 @@ web3 = { version = "0.18", default-features = false }
 [dev-dependencies]
 flate2 = "1.0"
 regex = "1.5"
+sqlx = { version = "0.6", default-features = false, features = ["runtime-tokio-native-tls"] }
```

Comment:
Context?

Reply:
Its needed so `cargo test -p shared` works, otherwise you get:
```
error: one of the features ['runtime-actix-native-tls', 'runtime-async-std-native-tls', 'runtime-tokio-native-tls', 'runtime-actix-rustls', 'runtime-async-std-rustls', 'runtime-tokio-rustls'] must be enabled
  --> /Users/nlordell/.cargo/registry/src/github.com-1ecc6299db9ec823/sqlx-rt-0.6.0/src/lib.rs:9:1
   |
9  | / compile_error!(
10 | |     "one of the features ['runtime-actix-native-tls', 'runtime-async-std-native-tls', \
11 | |      'runtime-tokio-native-tls', 'runtime-actix-rustls', 'runtime-async-std-rustls', \
12 | |      'runtime-tokio-rustls'] must be enabled"
13 | | );
   | |_^

error: could not compile `sqlx-rt` due to previous error
```